src_extra/networklayer_tcp.c

```diff
@@ -35,7 +35,6 @@
 
 /* workaround a glibc bug where an integer conversion is required */
 #if !defined(_WIN32)
-# include <features.h>
```

Comment:
features.h defines __GNU_LIBRARY__ et al.
We need to detect the glibc version, since the -Wconversion flag uncovers a bug in old glibc headers.

So far I am unaware of a non-Windows libc that does not come with this header.
Even embedded-stuff like musl has it: https://github.com/rofl0r/musl/blob/master/include/features.h

@Pro No features.h on Mac??? That's odd. What compiler do you use?